starport/pkg/numbers/numbers_test.go

```diff
@@ -7,6 +7,37 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestParseListRange(t *testing.T) {
+	cases := []struct {
+		list   string
+		parsed []uint64
+	}{
+		{"1,2,3", []uint64{1, 2, 3}},
+		{"1, 2,3 ", []uint64{1, 2, 3}},
+		{",1, 2,", []uint64{1, 2}},
+		{"1-3 ", []uint64{1, 2, 3}},
+		{"1-3,8 ", []uint64{1, 2, 3, 8}},
+		{"1-3,8-11 ", []uint64{1, 2, 3, 8, 9, 10, 11}},
+		{"1-3,8-11,33 ", []uint64{1, 2, 3, 8, 9, 10, 11, 33}},
+		{"1-3,8-11,33-36 ", []uint64{1, 2, 3, 8, 9, 10, 11, 33, 34, 35, 36}},
+		{",", []uint64{}},
+		{",-", []uint64{}},
+		{",10-", []uint64{10}},
+		{"10-", []uint64{10}},
+		{"-10", []uint64{10}},
+		{"10-10", []uint64{10}},
+		{"12-8", []uint64{8, 9, 10, 11, 12}},
+		{"12-8,4-1", []uint64{8, 9, 10, 11, 12, 1, 2, 3, 4}},
```

Comment:
What will happen if two ranges overlap each other e.g. "1-4,3-5" or in case of just duplication 1,2,2. Will it be handled correctly on the higher level of abstraction?

Reply:
Great, we got a bug. I made the fix on the commit https://github.com/tendermint/starport/pull/1809/commits/b6ee456a4f7638fa0a4ea4302989bd6f2afa6fdd